CONTRIBUTING.md

```diff
@@ -139,6 +143,83 @@ The description field should
 - be neutral and not use biased additions like ~~leading~~, ~~innovative~~, ~~flexible~~ or buzzwords (~~performance~~, ~~next generation~~)
 - not contain hyperlinks or markup
 
+
+### Compatibility information
+
+FMU exporting and importing tools provide information to their users, how they have validated their FMI support.
+Providing and maintaining this information is in the responsibility of the tool vendors; in the tools page of the FMI Webpage a link to a repository or webpage of the tool vendor will be provided. Tool vendors are encouraed to clearly mark this webpage as provided by them and not Modelica Association or the FMI Project e.g. by prominently placing their logo on this page.
+
+#### Exporting tools
+
+* List the importing tools (including version information), with which the FMUs from this exporter have been imported and simulated successfully.
+This could be (free or commercial) tools available to the tool vendor of the exporting tool. Or the tests could be performed by third parties (users, tool vendords of importing tools). 
```

Comment:
```suggestion
This could be (free or commercial) tools available to the tool vendor of the exporting tool. Or the tests could be performed by third parties (users, tool vendors of importing tools). 
```